DataFormats/PatCandidates/interface/VIDCutFlowResult.h

```diff
@@ -0,0 +1,60 @@
+#ifndef __DataFormats_PatCandidates_VIDResult_H__
+#define __DataFormats_PatCandidates_VIDResult_H__
+
+#include <unordered_map>
```

Comment:
Ah ok, I can just move to std::map instead of unordered_map. Then revert later with an i/o rule.